src/algebraic_topology/simplicial_set.lean

```diff
@@ -43,14 +45,17 @@ namespace sSet
 is the Yoneda embedding of `n`. -/
 def standard_simplex : simplex_category ⥤ sSet := yoneda
 
-localized "notation `Δ[`n`]` := standard_simplex.obj (n : ℕ)" in sSet
+localized "notation `Δ[`n`]` := standard_simplex.obj (simplex_category.mk n)" in sSet
```

Comment:
Could we change this locale name to `simplicial`, to avoid needing to open two?

Reply:
Good point. Done!